bloomrun.js

```diff
@@ -16,6 +16,7 @@ function BloomRun (opts) {
   this._isDeep = opts && opts.indexing === 'depth'
   this._buckets = []
   this._properties = new Set()
+  this._defaultResult = null
```

Comment:
Not true, a regex is only applicable when it matches right? If it never matches it is not applicable. There is a distinction between catch all and match all. 

By it's very nature you can only have one catch all. The idea being when Nothing matches then I use my catch all